requirements.txt

```diff
@@ -5,3 +5,7 @@ GPUtil
 kazoo
 pymysql
 pycrypto
+encryption
+hashlib
```

Comment:
I'm getting errors installing this, and it seems extraneous if python has hashing functions built-in.

`print "unknown OS, please update setup.py
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("unknown OS, please update setup.py")?`

Is this module python2 exclusive/still maintained?